Description: get MAX_PATH from OS defined PATH_MAX
 original source defines MAX_PATH to 512
 fortified source build aborts because of that
Forwarded: yes

--- vfu-4.18.orig/vslib/vsuti.h
+++ vfu-4.18/vslib/vsuti.h
@@ -25,6 +25,10 @@
 #include <pwd.h>
 #include <sys/types.h>
 #include <sys/param.h>
+#ifdef _TARGET_LINUX_
+#include <linux/limits.h>
+#define MAX_PATH PATH_MAX
+#endif
 #endif
 
 #include <assert.h>
